src/Microdown-DocumentBrowser/MicIntermediateBlock.class.st

```diff
@@ -1,6 +1,10 @@
 "
 I am a block that is used to group other nodes internally in a microdown trees.
-I am not used by the parser, but by post-processing tools.
+I am not used by the parser, but by post-processing tools. hjkhjhkjhkjhaksjdhakjdhaskjhakfjh\
```

Comment:
What is this?

Reply:
This is nothing apparently I modified the class comments demonstrating the documentation browser when editing a class comment.